### PR TITLE
Doc: Changes to the import of a Windows VM

### PR DESCRIPTION
**Pull request by mseralessandri**
I slightly reworded the section related to the import of a Windows VM from a foreign hypervisor to better clarify that the steps described are optional

### CODE REVIEW

**Comment by simondeziel** [comment]
As mentioned in the Google Meet, the MD exceptions are not very smart and we have to hardcode which line numbers to ignore in a given file. Since you've changed the number of lines in a file that required some exceptions, the exception file needs an update as well. In `doc/.sphinx/.markdownlint/exceptions.txt`, you'll find:

```
.tmp/doc/howto/import_machines_to_instances.md:114: MD034 Bare URL used
.tmp/doc/howto/import_machines_to_instances.md:218: MD034 Bare URL used
```

I suspect that subtracting "1" to those 2 line offsets will do the job.

File: doc/howto/import_machines_to_instances.md [hunk 1]
```diff
@@ -47,8 +47,7 @@ The tool can also inject the required VIRTIO drivers into the image:
    * Download [`rhsrvany.exe` and `pnp_wait.exe`](https://github.com/rwmjones/rhsrvany?tab=readme-ov-file#binary-releases), and place them in the `/usr/share/virt-tools/` directory.
 
    ````{tip}
-   If you want to convert a Windows VM from a foreign hypervisor manually,
-   you must install both the required Windows drivers (as described above) and `virt-v2v` (>= 2.3.4).
+   The `lxd-migrate` command with the `--conversion=format,virtio` option automatically converts the image and injects the VIRTIO drivers during the conversion. However, if you want to manually convert a Windows VM from a foreign hypervisor, you must install both the required Windows drivers (as described above) and `virt-v2v` (>= 2.3.4).
```
Comment (markylaing) [inline doc/howto/import_machines_to_instances.md:50]: Can you please add a new line after each sentence? Thanks :)

**Review by markylaing — CHANGES_REQUESTED** [review]
Could you please split this into two commits? One for updating the doc, and another for updating the linter exceptions?

Also, can the commit message for updating the documentation please be prefixed with just the directory? This is to keep the commit message a bit shorter (and also because the file that has been changed is clear from the commit). I suggest:
```
doc/howto: Update windows conversion tip to mention new `lxd-migrate` functionality
```

**Comment by simondeziel** [comment]
@mseralessandri can you rebase and drop the first commit? Now that the doc changed 2 lines for 2 lines, the MD exception line offsets are valid again as they were initially so don't need to be changed anymore.

**Review by simondeziel — APPROVED** [review]

**Review by tomponline — APPROVED** [review]

**Comment by tomponline** [comment]
@mseralessandri please can you sign your commits with your ssh key https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#ssh-commit-signature-verification

**Comment by simondeziel** [comment]
> @mseralessandri please can you sign your commits with your ssh key https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#ssh-commit-signature-verification

Those doc's have everything but to save you some time, here are the 3 configs needed:

```
$ git config --global gpg.format ssh
$ git config --global commit.gpgsign true
$ git config --global user.signingkey "~/.ssh/id_ed25519.pub"  # Adapt if you don't use ED25519 key
```

Then you need to add a **signing** key to your GitHub account settings in https://github.com/settings/keys. There you'll need to add the SSH pub key (the content of the `.pub` file) you used as signing key above.

**Pull request merged**